src/interfaces/IPublicAllocator.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.21;
+
+import {IMetaMorpho, IMorpho, MarketAllocation, Id} from "../../lib/metamorpho/src/interfaces/IMetaMorpho.sol";
+
+struct FlowCap {
+    /// @notice The maximum allowed inflow in a market
+    uint128 maxIn;
+    /// @notice The maximum allowed outflow in a market
+    uint128 maxOut;
+}
+
+struct FlowConfig {
```

Comment:
```suggestion
struct FlowCapsConfig {
```

Reply:
FlowCapsConfig.flowCaps vs FlowCapsConfig.caps vs FlowConfig.cap, at least the first seems too long.

Reply:
What about

- `FlowConfig { Id id; FlowRange range }`
- `SupplyConfig { Id id; uint cap }`

(or for me `FlowCap` being 2 fields is fine)